files/fr/web/html/global_attributes/enterkeyhint/index.html

```diff
@@ -0,0 +1,96 @@
+---
+title: enterkeyhint
+slug: Web/HTML/Global_attributes/enterkeyhint
+browser-compat: html.global_attributes.enterkeyhint
+translation_of: 'Web/HTML/Global_attributes/enterkeyhint'
+---
+<div>{{HTMLSidebar("Global_attributes")}}</div>
+
+<p><a href="/fr/docs/Web/HTML/Global_attributes">L'attribut universel</a> <strong><code>enterkeyhint</code></strong> 
+est un attribut à valeur contrainte définissant le libellé d'action (ou d'icône) à présenter pour la clé saisie sur le clavier virtuel.</p>
```

Comment:
"touche entrée" instead of "clé saisie"

Reply:
"ou l'icône" instead of "ou d'icône"

Reply:
fixed in https://github.com/mdn/translated-content/pull/1823/commits/047ee2186aa63fefe5382046869d2d4fcd5338ad